src/main/java/de/tk/opensource/privacyproxy/routing/RoutingHandler.java

```diff
@@ -120,41 +168,39 @@ public ResponseEntity<Object> handlePostInternal(
 				httpRequest.addHeader(header.getKey(), header.getValue());
 			}
 
-			LOGGER.debug("Calling {}", httpRequest.getURI());
+			logger.debug("Calling {}", httpRequest.getURI());
 
 			// send it
 			final HttpResponse response = httpClient.execute(httpRequest);
 
 			// add response headers
-			final HttpHeaders responseHeaders = new HttpHeaders();
-			responseHeaders.add(HttpHeaders.CACHE_CONTROL, "no-cache");
-
-			// add provider specific response headers
-			for (final String headerName : getWhitelistedResponseHeaders()) {
-				final String headerValue = response.getFirstHeader(headerName).getValue();
-				if (headerValue != null) {
-					responseHeaders.add(headerName, headerValue);
-				}
-			}
+			final HttpHeaders responseHeaders =
```

Comment:
smart geschachtelt, aber nicht mehr lesbar. Bitte auslagern in Methode und Variablen/Konstanten spendieren. Kommentar dann auch hinfällig.